scripts/compose.py

```diff
@@ -1027,7 +1027,12 @@ class Filebeat(BeatMixin, StackService, Service):
 
     def __init__(self, **options):
         super(Filebeat, self).__init__(**options)
-        config = "filebeat.yml" if self.at_least_version("6.1") else "filebeat.simple.yml"
+        if self.at_least_version("7.2"):
+            config = "filebeat.yml"
+        elif self.at_least_version("6.1"):
+            config = "filebeat_71.yml"
```

Comment:
I think it's confusing to call this 7.1, perhaps just `filebeat.6.x-compat.yml`?

Reply:
sure